inst/tests/test-template.R

```diff
@@ -0,0 +1,61 @@
+context("templates")
+
+# Searches for an html dependency of format name[version], as in "d3[3.5.10]",
+# within the html-dependencies script tag
+findDep <- function(x, name, version) {
```

Comment:
Maybe place in `helper-findDep.R`? Are you anticipating any re-use in future tests?